rm-ethereum.conf

```diff
@@ -0,0 +1,14 @@
+{
+"gasPrice":"1000000000",
+"maxgasLimit":"1000000000",
+"debugInfo":"false",
+"rpcAddress":"http://localhost:8545",
+"channelAddr":"0x0f1736FA3eB54BD8C234725BD658201DA0B4190D",
+"tokenAddr":"0x0fC373426c87F555715E6fE673B07Fe9E7f0E6e7",
```

Comment:
Is this our deployed TGE contract on the Kovan network? Or one of David's test contracts?

Reply:
This is the David's test contracts yet.

Reply:
The tokenContract and channelContract are two different things.

Reply:
Nevermind I get what you mean lol. I haven't had enough coffee yet.